Governance.md

```diff
@@ -0,0 +1,323 @@
+-   make decisions about strategic collaborations with other organizations or
+    individuals
+-   make decisions about specific technical issues, features, bugs, and Pull
+    Requests
+-   make decisions when regular community discussion doesn't produce consensus
+    on an issue in a reasonable time frame
+-   update policy documents such as this one
+
+However, the Council's primary responsibility is to facilitate the ordinary
+community-based decision-making procedure described above. If we ever have to
+step in and formally override the Community for the health of the Project, then
+we will do so, but we will consider reaching this point to indicate a failure in
+our leadership.
+
+#### Council Membership
+
+The Steering Council is composed of up to 5 Members that serve for 2-year terms.
+Council Members are nominated by Project Maintainers or themselves. Project
+Maintainers will vote on nominees during a voting period of at least one week
+that is announced sufficiently far in advance (e.g., 1 month) to ensure that all
+Project Maintainers that wish to vote are able to do so. No quorum of Project
+Maintainers is necessary for this voting process.
+
+The Council will be initially formed from the set of existing Project
+Maintainers who, as of mid-2022, have been significantly active over the last
+year.
+
+To become eligible to join the Steering Council, an individual must be a Project
+Contributor who has produced contributions that are substantial in quality and
+quantity, and sustained over a period of time sufficient to demonstrate their
+understanding of the needs of the project and community. These contributions may
+include, but are not limited to code, code review, participation in GitHub
+Issues, Pull Requests, and Discussions, community help/building, education,
+outreach, and other activities that benefit the Project and the Community. We
+want to encourage a diverse array of backgrounds, talents, and perspectives on
+the Council, which is why we explicitly do not define code or code-related
+metrics as the sole metric on which Council membership will be evaluated.
+
+Any former Council member can be considered for membership again at any time in
+the future, like any other Project Contributor; there is no maximum term length.
+
+If a Council member becomes inactive in the project for a period of one year,
+they will be considered for removal from the Council. Before removal, they will
+be contacted to determine if they plan on returning to active participation. If
+not, they will be removed immediately upon a vote of the remaining Council
+members, and their vacancy will be filled in the process outlined above.
+
+The Council reserves the right to eject current Members if they are deemed to be
+actively harmful to The Project's well-being, and attempts at communication and
+conflict resolution have failed. This requires the consensus of the remaining
```

Comment:
Just to be sure, is this "consensus" in the same definition as the technical decision-making process as described above? That is, a Member could veto an ejection of another Member?

Reply:
Implicitly, probably yes. 

How about 

```Markdown
This requires the consensus (as described above) of the remaining Members.
```